src/main/java/gibson/task/Task.java

```diff
@@ -0,0 +1,46 @@
+package gibson.task;
+
+/**
+ * Represents a task with a task string that describes a job to be done
+ */
+public class Task {
+    private String taskString;
+    private boolean done;
+
```

Comment:
Considering it is a boolean, do you think naming it as `isDone` (or something similar) would be more in accordance with the [coding standards](https://se-education.org/guides/conventions/java/intermediate.html)? The same goes for other such boolean names as well.